src/js/components/Tag/__tests__/__snapshots__/Tag-test.tsx.snap

```diff
@@ -551,6 +551,7 @@ exports[`Tag onRemove 1`] = `
       </span>
     </div>
     <button
+      aria-label="Remove undefined Test"
```

Comment:
If name doesn't exist, I wouldn't expect it to include "undefined" in the aria-label. I'd expect this to read "Remove Test"